packages/react/src/ActionBar/ActionBar.tsx

```diff
@@ -160,6 +161,9 @@ const overflowEffect = (
     }
 
     updateListAndMenu({items, menuItems})
+  } else if (numberOfItemsPossible >= childArray.length && hasActiveMenu) {
```

Comment:
nit: the '=' part of this condition(I think) never happens because of the equality check on L139

Reply:
Thank you! the less code the better 😂